api/envoy/extensions/resource_monitors/downstream_connections/v3/downstream_connections.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package envoy.extensions.resource_monitors.downstream_connections.v3;
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.resource_monitors.downstream_connections.v3";
+option java_outer_classname = "DownstreamConnectionsProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/resource_monitors/downstream_connections/v3;downstream_connectionsv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Downstream connections]
+// [#extension: envoy.resource_monitors.downstream_connections]
+
+// The downstream connections resource monitor tracks the global number of open downstream connections.
+// [#not-implemented-hide:]
+message DownstreamConnectionsConfig {
+  // Maximum threshold for global open active downstream connections, defaults to 0.
+  // If monitor is configured via Overload manager api and has no value set, Envoy will reject all incoming connections.
```

Comment:
I think this is up for discussion. Having it '>0' would require users to explicitly configure threshold for monitor (and fail/error on startup otherwise if threshold is not configured). While '>=0' can be more tricky for users, if they forget to configure the threshold, monitor will use default value 0 and reject all incoming connections. First option (require explicit value > 0) is cleaner in my opinion, although monitor will not be able to reject all incoming connections.

Reply:
sgtm